### PR TITLE
PMM-4767 PMM-4907 Add AWS instance check page.

### PR DESCRIPTION
**Pull request by AlekSi**
https://github.com/Percona-Lab/pmm-submodules/pull/554

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/pmm-server?pullRequest=181) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/pmm-server?pullRequest=181) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: AlekSi<br/>:x: lunaticusgreen<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/pmm-server?pullRequest=181) it.</sub>

File: installation-wizard-page/src/App.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { message } from "antd";
+import axios from "axios";
```
Comment (askomorokhov) [inline installation-wizard-page/src/App.js]: Personally I prefer to use a single quote `'`.
But exist different code convention.
https://github.com/airbnb/javascript
https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines
https://eslint.org/docs/rules/jsx-quotes
https://github.com/airbnb/javascript/tree/master/react#quotes
We need to choose one to use everywhere.

File: installation-wizard-page/src/App.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        message.error(
+          error.response.data.message
+            ? error.response.data.message
+            : error.message
+        );
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  };
```
Comment (askomorokhov) [inline installation-wizard-page/src/App.js]:
```suggestion
const checkInstance = async () => {
     setLoading(true);
     try {
      const response = await axios.post("/v1/AWSInstanceCheck", { instance_id: instanceId });
      // handle success
      window.location.href = "/";
     } catch (error) {
         // handle error
         message.error(error.response.data.message ? error.response.data.message : error.message);
     } finally {
     () =>setLoading(false);
     }
   };
```

IMHO, if lambda is `async` I'd preferred to use async/await in whole code. It looks more compact.
What do you think?

File: installation-wizard-page/src/App.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+              onClick={checkInstance}
+              type="primary"
+              loading={loading}
+              disabled={loading}
+              className={"instance-id-submit-button"}
+            >
+              Submit
+            </Button>
+          </p>
+          <a href="/">Where should i get my instance ID</a>
```
Comment (askomorokhov) [inline installation-wizard-page/src/App.js]:
```suggestion
          <a href="/">Where should I get my instance ID?</a>
```
Is it correct link href?

File: installation-wizard-page/src/App.js (hunk 1 above)
Comment (lunaticusgreen) [reply installation-wizard-page/src/App.js]: You're right, I'll find appropriate code guide for us (with linter rules too) and will add it to all our react repos

**Pull request merged**